src/tests/Parser/Priest/Discipline/Normalizers/AtonementSuccessiveDamage.test.js

```diff
@@ -0,0 +1,106 @@
+import AtonementSuccessiveDamage from 'Parser/Priest/Discipline/Normalizers/AtonementSuccessiveDamage';
+import SPELLS from 'common/SPELLS';
+
+import {
+  DamagingEvent1,
+  DamagingEvent2,
+  DamagingEvent3,
+  AtonementOnSelf1,
+  AtonementOnSelf2,
+  AtonementOnPlayer1,
+  AtonementOnPlayer2,
+  AtonementOnPlayer3,
+} from './Fixtures/TestingEvents';
+
+describe('DisciplinePriest.Reordering', () => {
+
+  let atonementSuccessiveDamageNormalizer;
+  beforeEach(() => {
+    atonementSuccessiveDamageNormalizer = new AtonementSuccessiveDamage({
+      reorder: () => true,
+      toPlayer: () => true,
+      byPlayer: () => true,
+      toPlayerPet: () => false,
+      byPlayerPet: () => false,
+    });
+  });
+
+  it('If 2 damaging events happen simultaneously, the atonement ahead is split in two', () => {
+
+    const AtonementOf2DamingEventsGroupedTogether = [
+      AtonementOnPlayer2,
+      DamagingEvent1,
+      DamagingEvent2,
+      AtonementOnPlayer1,
+      AtonementOnPlayer2,
+      AtonementOnPlayer1,
+      AtonementOnPlayer2
+    ];
+
+    let result = atonementSuccessiveDamageNormalizer.normalize(AtonementOf2DamingEventsGroupedTogether);
+    expect(result[0]).toBe(AtonementOnPlayer2)
```

Comment:
I think this would be easier to read if you check that it matches the order of an array instead;

```
expect(result).toEqual([
  AtonementOnPlayer2,
  DamagingEvent1,
  ...etc
]);
```

Reply:
Agree with Zero, you might find you have to use `toDeepEqual` to compare though and not `toEqual`.

Reply:
I reckon toEqual does the exact same check as what happens here

Reply:
toEqual worked